src/validation.h

```diff
@@ -656,6 +655,20 @@ class CChainState
         return m_coins_views->m_dbview;
     }
 
+    //! @returns A reference to the mempool.
+    CTxMemPool* GetMempool()
+    {
+        return m_mempool;
+    }
+
+    /**
+     * Update the mempool
+     */
+    CTxMemPool* SetMempool(CTxMemPool* mempool)
```

Comment:
Not a fan of adding test-only helpers to the real code. Might be better to move this to the test code.

Reply:
Good catch. Done in 3f7d1c1